### PR TITLE
Added languages support page

### PR DESCRIPTION
**Pull request by davidtsong**
Rough draft - waiting to get feedback on content and formatting

### CODE REVIEW

**Comment by davidtsong** [comment]
@adamschwartz 
Screen Shot 2020-07-27 at 3 50 58 PM
image
I need some help formatting the sub link properly. Will add the rest of the content links to external pages afterwards.

**Comment by davidtsong** [comment]
image (1)

File: src/content/reference/platform/language.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
+# Languages
+
+## Background
+
+Workers is a polyglot platform. You can likely write code on Workers with a language you already know. We originally built Workers to use JS/TS and over time we've added support for new languages. We plan to add popular languages like Python and continue to deepen support for added languages.
```
Comment (koeninger) [inline src/content/reference/platform/language.md:6]: "plan to add popular languages like python" is in contrast to the serverless week blog posts "excited to announce support for Python"

I wouldn't say we originally built workers to use JS/TS, I'd say we built it to use JavaScript and WebAssembly

File: src/content/reference/platform/language.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+</TableWrap>
+
+### See Also:
+- [`Serverless Rust with Cloudflare Workers`](https://blog.cloudflare.com/cloudflare-workers-as-a-serverless-rust-platform/)
+- [`WebAssembly on Cloudflare Workers`](https://blog.cloudflare.com/webassembly-on-cloudflare-workers/)
+
+## Compiled to JavaScript
+
+Since Workers runs off of JS, you can also program on Workers with any language than can compile down to JS including the languages below. Learn more by checking out the example projects.
```
Comment (koeninger) [inline src/content/reference/platform/language.md:38]: I think it's worth emphasizing Typescript type definitions https://github.com/cloudflare/workers-types  here

**Comment by davidtsong** [comment]
Screen Shot 2020-07-29 at 3 49 27 PM

**Comment by davidtsong** [comment]
Screen Shot 2020-07-29 at 3 53 22 PM

File: src/content/reference/platform/language.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+### See Also:
+- [`TypeScript Type Definitions`](https://github.com/cloudflare/workers-types)
+- [`Hello World for Workers in JS`](https://github.com/cloudflare/worker-template)
+- [`Javascript and Webstandard APIs`](/reference/runtime-apis/web-standards)
+- [`Starters - Code snippets`](/starters)
+- [`Tutorials`](/tutorials)
+
+## WASM-supported
+
+WebAssembly -- often abbreviated as "WASM" -- is a technology that extends the web platform to support compiled languages like C, C++, Rust, Go, and more. Since these languages can be compiled to a special WASM binary format and then loaded in a browser, you can use them on the Workers platform!  Learn more by checking out the example projects.
```
Comment (koeninger) [inline src/content/reference/platform/language.md:30]: I think mentioning Go is misleading given that we have no examples in our docs.

Comment (davidtsong) [reply src/content/reference/platform/language.md:30]: yeah that makes sense, will remove it

**Comment by adamschwartz** [comment]
@davidtsong I’ll take a design pass at this now.

**Comment by adamschwartz** [comment]
Reverting since it ended up in old `/reference` directory, and added via https://github.com/cloudflare/workers-docs-engine/commit/0fc7995e98c7fa6a4030b5d0cc92589c0398e116 instead.

**Comment by adamschwartz** [comment]
Live: https://workers.cloudflaredocs.workers.dev/platform/languages

**Pull request merged**